system/include/emscripten/em_asm.h

```diff
@@ -0,0 +1,84 @@
+#ifndef __em_asm_h__
+#define __em_asm_h__
+
+
+#define CONCAT(x, y) CONCAT_(x, y)
```

Comment:
i still have no idea how this works ;) but why is this line needed? seems like it does nothing.